dealer.c

```diff
@@ -863,29 +924,78 @@ int shuffle (deal d) {
         break;
     }
   } else {
-    /* Algorithm according to Knuth. For each card exchange with a random
-       other card. This is supposed to be the perfect shuffle algorithm. 
-       It only depends on a valid random number generator.  */
-    for (i = 0; i < 52; i++) {
-      if (stacked_pack[i] == NO_CARD) { 
```

Comment:
...Is stacked_pack still used anywhere? If not, get rid of it, too.

Reply:
No, it's still used in Francois' exhaustive-search stuff. Leave it alone for now.